src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -882,6 +876,13 @@ def _maybe_store_problem(self):
         if not parent_span:
             return
 
+        # Track how many N+1s we would have detected, but couldn't, because of a truncated query.
```

Comment:
Nit, are these N+1's that we "would have detected"? or more so a count of potential N+1's that we can't validate because of truncation.

Reply:
:+1: New comment:
> Track how many N+1-looking problems we found but dropped because we couldn't be sure (maybe the truncated part of the query differs).

Does that help?

Reply:
Yeah, thanks!